Assets/VRTK/Prefabs/SnapDropZone/VRTK_SnapDropZone.cs

```diff
@@ -507,11 +338,11 @@ protected virtual void SetObjectHighlight()
             }
         }
 
-        protected virtual void ToggleHighlightColor()
+        public virtual void ToggleHighlightColor()
```

Comment:
public methods need to be grouped with other public methods above all protected methods, and public methods require documentation.